### PR TITLE
Fixed some issues

### PR DESCRIPTION
**Pull request by zaffnet**
Some small fixes. Reasons:

 In file [beginner_source/blitz/autograd_tutorial.py](beginner_source/blitz/autograd_tutorial.py)
The two sentences are rendered on the same line. See [this](https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html#gradients)

In file [beginner_source/blitz/cifar10_tutorial.py](beginner_source/blitz/cifar10_tutorial.py)
Grammatical error

In file [beginner_source/blitz/neural_networks_tutorial.py](beginner_source/blitz/neural_networks_tutorial.py)
"you can only specify a single number" may imply it is the only way to do it

### CODE REVIEW

File: beginner_source/blitz/neural_networks_tutorial.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __init__(self):
     def forward(self, x):
         # Max pooling over a (2, 2) window
         x = F.max_pool2d(F.relu(self.conv1(x)), (2, 2))
-        # If the size is a square you can only specify a single number
+        # If the size is a square you can specify a single number also
```
Comment (JoelMarcey) [inline beginner_source/blitz/neural_networks_tutorial.py]: This would change the meaning. @soumith @orionr -- is this correct?

**Comment by holly1238** [comment]
Closing this PR. All updates in this PR have already been fixed at some point.
